bin/batnode-sample.js

```diff
@@ -5,17 +5,18 @@
 const bat_sample = require('commander');
 const chalk = require('chalk');
 
-// const BatNode = require('../batnode').BatNode;
-const BatNode = require('../kad-bat-plugin/batnode.js').BatNode;
 const PERSONAL_DIR = require('../utils/file').PERSONAL_DIR;
 const HOSTED_DIR = require('../utils/file').HOSTED_DIR;
 const fileSystem = require('../utils/file').fileSystem;
 const fs = require('fs');
 
+const BatNode = require('../kad-bat-plugin/batnode.js').BatNode;
```

Comment:
Hm, one of the differences with my branch is I was working with the `/batnode.js` and not `/kad-bat-plugin/batnode.js`. I'm not sure having multiple version of BatNode going forward will help us. Since `/kad-bat-plugin/batnode.js` seems to work here, I think it might help if we copy it's contents, copy and paste over what's in `/batnode.js`. Then moving forward there won't be confusion about which `batnode` file to reference. How does that sound?

Reply:
In "index.js", we actually use the global `batnode.js` (the one in home dir). 
I actually test the global one in `batchain-sample.js` and the audit still works, would you suggest I should change this require file back to the global one?
It shouldn't create errors with the updated `batnode.js`.

Reply:
Huh cool I guess that wasn't my issue. Yeah if works with the global one then it seems like we should point the `batchain-sample` file and related testing nodes to the global one. I really want to delete `kad-bat-plugin.js` haha, but we don't necessarily need to do it in this PR - not actively using it is probably good enough.

Reply:
Once that's done it'll be good to merge for me.

Reply:
Require batnode.js in home dir: 2a78289a2c176db84047bda4f9cec060fef76a78